data/json/mapgen/mine/mine_spiral.json

```diff
@@ -0,0 +1,190 @@
+        "########@ @@@@@@@@@@@@@@@@@@ @######@                                  @",
+        "########@                    @######@                                  @",
+        "########@@@@@@@@@@@@@@@@@@@@@@######@                                  @",
+        "####################################@                                  @",
+        "####################################@@@@@@@@@@@@@@@@@@@@@@@            @",
+        "##########################################################@            @",
+        "##########################################################@            @",
+        "#####@@@@@@@@@@@@@@@@@@####@@@@@@@@@@@@@@@@@@@############@            @",
+        "#####                  ####                  @############@            @",
+        "#####       ####       ####       ####        ############@            @",
+        "#####%%%%%%######%%%%%%####!!!!!!######$$$$$$ ############@            @",
+        "#####%     ######     %####!     #####      $ ############@            @",
+        "#####% %%%%######%%%% %####! !!!!##### $$$$ $ ############@            @",
+        "#####% %  %######%  % %####! !  !##### $ >$ $ ############@            @",
+        "#####%    %######%    %####!    !##### $    $ ############@            @",
+        "#####%%%%%%######%%%%%%####!!!!!!##### $$$$$$ ############@            @",
+        "####       ######       ##       #####        ############@            @",
+        "###        ######                @####@      @############@            @",
+        "#@  @@@@@@@######@@@@@@@@@@@@@@@@@####@@@@@@@@############@            @",
+        "#@  @#####################################################@            @",
+        "#@  @#######################################################           @",
+        "#@  @@@@@@@######@@@@@@@@@@@@@@@@######@@@@@@@@@@@@@@@@@@@###          @",
+        "###        ######                ######                    ##          @",
+        "####       ######       ##       ######        ###        ####        ##",
+        "#####      ######      ####      ######       #####      ######      ###",
+        "#####^^^^^^######^^^^^^####&&&&&&######&&&&&&######******######******###",
+        "#####^     ######     ^####&     ######     &######*     ######     *###",
+        "#####^ ^^^^######^^^^ ^####& &&&&######&&&& &######* ****######**** *###",
+        "#####^ ^  ^######^  ^ ^####& &  &######&  & &######* *  *######*  * *###",
+        "#####^    ^######^    ^####&    &######&    &######*    *######*    *###",
+        "#####^^^^^^######^^^^^^####&&&&&&######&&&&&&######******######******###",
+        "#####       ####       ####       ####       ######      ######      ###",
+        "#####                  @##@                  @####@                  ###",
+        "#####@@@@@@@@@@@@@@@@@@@##@@@@@@@@@@@@@@@@@@@@####@@@@@@@@@@@@@@@@@@@###"
+      ],
+      "terrain": {
+        "@": [ [ "t_rock", 4 ], [ "t_rock_floor", 1 ] ],
```

Comment:
So this is usually rock, but sometimes floor?

Reply:
Yeah, it's located only next to the rock walls and is intended to break artificial look of straight lines of walls.